src/dynamic_fee.rs

```diff
@@ -43,6 +43,18 @@ impl Miner {
                             ]
                         })
                     }
+
+                    "quicknode" => {
+                        json!({
+                            "jsonrpc": "2.0",
+                            "id": 1,
+                            "method": "qn_estimatePriorityFees",
+                            "params": {
+                                 "last_n_blocks": 100,
+                                 "account": "oreV2ZymfyeXgNgBdqMkumTqqAprVqgBWQfoYkrtKWQ",
```

Comment:
You'll want to include the `ore_cli::consts::BUS_ADDRESSES` as well.

Reply:
Ah thanks!

Presently, [Quicknode's API](https://www.quicknode.com/docs/solana/qn_estimatePriorityFees) only accepts a single "account".

Might be possible to make a array of jsonrpc payloads in the single query? 
```
                    "quicknode" => {
                        json!([{
                            "jsonrpc": "2.0",
                            "id": 1,
                            "method": "qn_estimatePriorityFees",
                            "params": {
                                 "last_n_blocks": 100,
                                 "account": "oreV2ZymfyeXgNgBdqMkumTqqAprVqgBWQfoYkrtKWQ",
                             }
                        },{
                            "jsonrpc": "2.0",
                            "id": 2,
                            "method": "qn_estimatePriorityFees",
                            "params": {
                                 "last_n_blocks": 100,
                                 "account": ore_cli::consts::BUS_ADDRESSES[0],
                             }
                        },
                        ...
                        ])
                    }
```

( admittedly I'm a bit of a noob in Rust 😆 )